runtime/src/main/java/org/corfudb/protocols/API.java

```diff
@@ -0,0 +1,1135 @@
+    }
+
+    public static Response getPingResponse(Header header, ServerError error) {
+        PingResponse pingResponse = PingResponse.getDefaultInstance();
+        return Response.newBuilder()
+                .setHeader(header)
+                .setError(error)
+                .setPingResponse(pingResponse)
+                .build();
+    }
+
+    public static Request getRestartRequest(Header header) {
+        RestartRequest restartRequest = RestartRequest.getDefaultInstance();
+        return Request.newBuilder()
+                .setHeader(header)
+                .setRestartRequest(restartRequest)
+                .build();
+    }
+
+    public static Response getRestartResponse(Header header) {
+        return getRestartResponse(header, getNoServerError());
+    }
+
+    public static Response getRestartResponse(Header header, ServerError error) {
+        RestartResponse restartResponse = RestartResponse.getDefaultInstance();
+        return Response.newBuilder()
+                .setHeader(header)
+                .setError(error)
+                .setRestartResponse(restartResponse)
+                .build();
+    }
+
+    public static Request getResetRequest(Header header) {
+        ResetRequest resetRequest = ResetRequest.getDefaultInstance();
+        return Request.newBuilder()
+                .setHeader(header)
+                .setResetRequest(resetRequest)
+                .build();
+    }
+
+    public static Response getResetResponse(Header header) {
+        return getResetResponse(header, getNoServerError());
+    }
+
+    public static Response getResetResponse(Header header, ServerError error) {
+        ResetResponse resetResponse = ResetResponse.getDefaultInstance();
+        return Response.newBuilder()
+                .setHeader(header)
+                .setError(error)
+                .setResetResponse(resetResponse)
+                .build();
+    }
+
+    public static Request getSealRequest(Header header, long newEpoch) {
+        SealRequest sealRequest = SealRequest.newBuilder()
+                .setEpoch(newEpoch)
+                .build();
+        return Request.newBuilder()
+                .setHeader(header)
+                .setSealRequest(sealRequest)
+                .build();
+    }
+
+    public static Response getSealResponse(Header header) {
+        return getSealResponse(header, getNoServerError());
+    }
+
+    public static Response getSealResponse(Header header, ServerError error) {
+        SealResponse sealResponse = SealResponse.getDefaultInstance();
+        return Response.newBuilder()
+                .setHeader(header)
+                .setError(error)
+                .setSealResponse(sealResponse)
+                .build();
+    }
+
+    public static CorfuMessage.ResponseMsg getVersionResponse(CorfuMessage.HeaderMsg headerMsg, VersionInfo vi){
+        final Gson parser = new GsonBuilder().create();
+        String payload = parser.toJson(vi);
```

Comment:
Is there any specific reason that we don't use `final Gson parser = new Gson()` here? I also noticed this patter in the current `JSONPayloadMsg` definition.